css/style.css

```diff
@@ -0,0 +1,506 @@
+    top: 0;
+    transform: rotate(90deg);
+  }
+
+  #header__toggle:checked ~ .header__menu {
+    visibility: visible;
+    right: 0;
+  }
+
+  .header__btn {
+    display: flex;
+    align-items: center;
+    position: fixed;
+    top: 20px;
+    right: 30px;
+    width: 35px;
+    height: 35px;
+    z-index: 1;
+    cursor: pointer;
+  }
+
+  .header__btn > span,
+  .header__btn > span::before,
+  .header__btn > span::after {
+    display: block;
+    position: absolute;
+    width: 100%;
+    height: 5px;
+    background-color: #0056EC;
+    transition-duration: 0.25s;
+  }
+
+  .header__btn > span::before {
+    content: '';
+    top: -9px;
+  }
+
+  .header__btn > span::after {
+    content: '';
+    top: 9px;
+  }
+
+  .header__menu {
+    display: block;
+    position: fixed;
+    visibility: hidden;
+    top: 0;
+    right: -100%;
+    width: 300px;
+    height: 100%;
+    margin: 0;
+    padding: 80px 0;
+    list-style: none;
+    transition-duration: .25s;
+  }
+
+  .menu {
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  .menu__item {
+    display: block;
+    padding: 12px 24px;
+    font-size: 20px;
+    font-weight: 600;
+    text-decoration: none;
+    transition-duration: .25s;
+  }
+
+  .content {
+    justify-content: center;
+    max-width: 519px;
+    height: 906px;
+    flex-flow: column-reverse;
+  }
+
+  .content__wrap {
+    height: 455px;
+    max-width: 612px;
+    margin: 0 auto;
+  }
+
+  .content__title {
+    margin: 0 auto;
+    max-width: 400px;
+    height: 170px;
+    font-size: 50px;
+    line-height: 55px;
+    text-align: center;
+    margin-bottom: 65px;
+  }
+
+  .content__desc {
+    margin: 0 auto;
+    max-width: 275px;
+    height: 57px;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    margin-bottom: 40px;
+  } 
+
+  .content__bt {
+    margin: 0 auto;
+  }
+}
+
+@media (max-width: 548px) {
+
+  .wrap__site {
+    max-width: 548px;
+  }
+  
+  .header__logo {
+    display: none;
+  }
+
+  .header__btn {
+    width: 25px;
+  }
+
+  .header__btn > span,
+  .header__btn > span::before,
+  .header__btn > span::after {
+    height: 3px;
+  }
+
+  .header__btn > span::before {
+    content: '';
+    top: -7px;
+  }
+
+  .header__btn > span::after {
+    content: '';
+    top: 7px;
+  }
+
+  .content {
+    justify-content: center;
+    max-width: 519px;
+    height: 709px;
+    flex-flow: column-reverse;
+    margin: 0 auto;
+  }
+
+  .content__img {
+    background-image: url(../img/mobile.png);
+    height: 239px;
+    width: 280px;
+    margin: 0 auto;
+  }
+
+  .content__wrap {
+    height: 455px;
+    max-width: 612px;
+  }
+
+  .content__title {
+    max-width: 267px;
+    height: 96px;
+    font-size: 30px;
+    line-height: 40px;
+    text-align: center;
+  }
+
+  .content__desc {
+    max-width: 275px;
+    height: 57px;
+    font-size: 16px;
+    line-height: 19px;
+    text-align: center;
+    margin-bottom: 38px;
+  }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/miniProject/pullRequest?prid=5593059)